tests/TestHelpers/GraphHelper.php

```diff
@@ -446,6 +446,33 @@ public static function getGroups(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $groupId
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getGroup(
```

Comment:
there a function name getGroup already may be. rebase and may be this function can be named as `getGroupByGroupID` and also name the other function as `getGroupByGroyupName`

Reply:
https://github.com/owncloud/ocis/blob/fe4195d69410c0a13164d96d8d0770bec893995f/tests/TestHelpers/GraphHelper.php#L459  we already have this function